app/assets/stylesheets/modules/full_text_highlight.scss

```diff
@@ -17,10 +15,9 @@ dt.prepared-search-container {
 
 dd.blacklight-full_text_tesimv {
   border-left: 1px solid $color-gray-80;
-  float: none;
   margin-left: 15px;
   margin-top: 6px;
-  @extend .col-12;
```

Comment:
This node already has a .col class on it, so all we need to do is set the width.